### PR TITLE
added who

### PR DESCRIPTION
**Pull request by js3263827**

### CODE REVIEW

File: _posts/0000-01-04-js3263827.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Han Shot First!
```
Comment (kayakkat) [inline _posts/0000-01-04-js3263827.md]: are you sure 🔫

Comment (kayakkat) [inline _posts/0000-01-04-js3263827.md]: might need 2 "!!"

**Review by kayakkat — CHANGES_REQUESTED** [review]
looks good @js3263827 ...I left a suggestion  but do what you want  👍

File: _posts/0000-01-04-js3263827.md (hunk 1 above)
Comment (js3263827) [reply _posts/0000-01-04-js3263827.md]: yep 1977

**Comment by js3263827** [comment]
Need another review

**Review by kayakkat — APPROVED** [review]

**Pull request merged**